app/controllers/api/v1/posts.js

```diff
@@ -1,21 +1,74 @@
 const db = global.db;
-const {Sequelize} = require('sequelize');
+const { Sequelize } = require("sequelize");
 const Op = Sequelize.Op;
 
-module.exports =  (router) => {
-
-  router.post('/', async (req, res) => {
+module.exports = (router) => {
+  router.post("/", async (req, res) => {
     req.body.userId = req.user.id;
     const post = await db.Posts.create(req.body);
-    res.json({post: post});
+    res.json({ post: post });
+  });
+
+  router.get("/", async (req, res) => {
+    const posts = await db.Posts.findAll({
+      include: [
+        {
+          model: db.Users,
+        },
+      ],
+    });
+    res.json({ posts: posts });
   });
 
-  router.get('/', async (req, res) => {
+  router.get("/me", async (req, res) => {
     const posts = await db.Posts.findAll({
```

Comment:
Same here optional pagination needs to be added

Reply:
I have updated this in the recent commit.